src/Microsoft.ML.Tokenizers/PreTokenizer/Whitespace.cs

```diff
@@ -4,21 +4,29 @@
 
 using System;
 using System.Collections.Generic;
+using System.Text.RegularExpressions;
 
 namespace Microsoft.ML.Tokenizers
 {
     /// <summary>
     /// The pre-tokenizer which split the text at the word boundary.
     /// The word is a set of alphabet, numeric, and underscore characters.
     /// </summary>
-    public sealed class WhiteSpace : PreTokenizer
+    public sealed partial class WhiteSpace : PreTokenizer
     {
         /// <summary>
         /// Gets a singleton instance of the WhiteSpace pre-tokenizer..
         /// </summary>
-        public static readonly WhiteSpace Instance = new WhiteSpace();
+        public static WhiteSpace Instance { get; } = new WhiteSpace();
```

Comment:
This is a public API change. We discourage public fields. Both here and on EnglishRoberta, I change the public static readonly field to a public static get-only property.